WeWatch/WeWatch/Preview Content/Assets/Font/Font extension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Font extension.swift
+//  WeWatch
+//
+//  Created by Anton on 25/11/2024.
+//
+
+import SwiftUI
+
+extension Font {
+    static var poppinsBold30px: Font { Font.custom("Poppins-Bold", size: 30)}
+    static var poppinsBold24px: Font { Font.custom("Poppins-Bold", size: 24)}
+    static var poppinsBold20px: Font { Font.custom("Poppins-Bold", size: 20)}
+    static var poppinsBold16px: Font { Font.custom("Poppins-Bold", size: 16)}
+    static var poppinsRegular16px: Font { Font.custom("Poppins-Regular", size: 16)}
+    static var poppinsRegular14px: Font { Font.custom("Poppins-Regular", size: 14)}
+    static var poppinsRegular13px: Font { Font.custom("Poppins-Regular", size: 13)}
```

Comment:
Add access modifiers. + format it in a better way as one-liners not the best choice. 

```
static var poppinsBold30px: Font { 
   .custom("Poppins-Bold", size: 30)
}
```